### ISSUE TITLE
fsockopen() is missing

### ISSUE THREAD

**Issue by simPod** [thecodingmachine/safe#81]
Right now, I have to do this:

```php
        $filePointer = fsockopen(...);
        if ($filePointer === false) {
            throw new SocketsException($errstr, $errno);
        }
```

It would be nice to do `$filePointer = \Safe\fsockopen(...);` instead.

```
 * @return resource fsockopen returns a file pointer which may be used
 * together with the other file functions (such as
 * fgets, fgetss,
 * fwrite, fclose, and
 * feof). If the call fails, it will return false
```

**Comment by moufmouf** [comment]
Absolutely! Let's add `fsockopen` to the list :)

**Comment by moufmouf** [comment]
Closed in #84.
